index.html

```diff
@@ -1,76 +1,44 @@
-    />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css"
-    />
-    <link rel="stylesheet" href="assets/css/style.min.css" type="text/css" />
 
-    <title>User Centric Frontend Development Resource Pack</title>
-  </head>
-  <body>
-        <div class="loader">Loading...</div>
-      <main>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="apple-touch-icon" sizes="180x180" href="assets/favicons/apple-touch-icon.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/favicons/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="assets/favicons/favicon-16x16.png" />
+  <link rel="icon" type="image/x-icon" href="assets/favicons/favicon.ico" />
+  <!-- Associated links for styles & frameworks -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.4.1/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" />
+  <link rel="stylesheet" href="assets/css/style.min.css" type="text/css" />
+
+  <title>User Centric Frontend Development Resource Pack</title>
+</head>
+
+<body>
+  <div class="loader">Loading...</div>
+  <main>
     <header>
       <!-- Nav -->
       <nav class="navbar navbar-expand-lg navbar-dark bg-custom fixed-top" id="navbar">
         <!-- Logo -->
         <a class="navbar-brand" href="index.html">
-          <img
-            class="image-fluid"
-            src="assets/images/slackIcon.png"
-            alt="slackIcon"
-            title="Slack Logo"
-          />
+          <img class="image-fluid" src="assets/images/slackIcon.png" alt="slackIcon" title="Slack Logo" />
         </a>
 
         <!-- Toggler Icon on Mobile -->
-        <button
-          class="navbar-toggler"
-          type="button"
-          data-toggle="collapse"
-          data-target="#navbarNavDropdown"
-          aria-controls="navbarNavDropdown"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
-        >
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown"
+          aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
 
         <!-- Navigation Links -->
         <div class="collapse navbar-collapse" id="navbarNavDropdown">
           <ul class="navbar-nav ml-auto uppercase">
             <li class="nav-item">
-              <a class="nav-link" href="#about"
-                >About<span class="sr-only">(current)</span></a
-              >
+              <a class="nav-link" href="#about">About<span class="sr-only">(current)</span></a>
```

Comment:
sr-only is not to be used with FA5 only for FA4 but we can change this at a later time.

Reply:
sr-only class wasn't a change I made. I just reformatted the code there.

Reply:
Right we can just resolve this and add it to be removed at a later time then